core/src/main/java/org/keycloak/representations/idm/RealmRepresentation.java

```diff
@@ -144,8 +152,11 @@
 
     // Client Policies/Profiles
 
-    protected ClientProfilesRepresentation clientProfiles;
-    protected ClientPoliciesRepresentation clientPolicies;
+    @JsonProperty("clientProfiles")
+    protected JsonNode clientProfiles;
+
+    @JsonProperty("clientPolicies")
+    protected JsonNode clientPolicies;
```

Comment:
I wonder whether we should have an explicit setter to be used by the Jackson mapper. This way we don't verify at this level that the provided profiles/policies are an actual and valid representation.